<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentGameDialog</class>
 <widget class="QWidget" name="PresentGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresentGameDialog</string>
  </property>
  <widget class="QPushButton" name="btn_Present">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подарить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_Game">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Game">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите игру:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_User">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите пользователя:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_User">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
